r_basics_introduction/r_basics_introduction.md

```diff
@@ -599,26 +639,28 @@ Which of the following statements are accurate?
 [[?]] Hint: multiple correct answers!
 
 <div class = "answer">
-
 <details><summary>Click to see an explanation of the answer.</summary>
 
 Functions can have zero, one, or many inputs.  An example of a function taking zero inputs is `getwd()`, which tells you what R considers its current "working directory".
 
 You only need to install a package using `install.packages()` **once** per computer (unless the package has a new update that you want to install).  You do, however, have to load a package using `library()` before its functions are available to you.  That means loading it each time you start a new session of R.  **When you knit a document, that counts as a separate session**, so make sure and put a `library()` in place for each package you use in your code in each R Markdown document you create.  Oh, and yep, for `install.packages()` you do indeed have to put quotes around the name of the package.
 
+</details>
+
 </div>
 </div>
 
 ### Hands-On: Import Data
```

Comment:
Do we potentially need to include more instruction before throwing them into this section, or is it fine? I'm mostly thinking about the fact that most of these exercises don't have anything to do with importing, they moreso have to do with clicking around in RStudio, exploring data, and noticing stuff in the data viewer. 

I can't remember how much of that sort of stuff they already saw us demo in the workshop by the time we send them to do this.

Reply:
I think this is a good catch.  I'll at the very least add a screenshot here with good alt text.

Reply:
Upon rereading this I think I missed your point but stumbled upon a new one, ha!  I think we are gentle enough in the introduction to data importing and in the interest of time I'll leave this be.  But I did add a bit of enrichment in the "if you have to install tidyverse on your own computer" callout box since many learners may be issuing their very first line of code....